.github/workflows/wheels.yml

```diff
@@ -28,8 +24,8 @@ jobs:
         os: [
           ubuntu-latest,
           windows-latest,
-          macos-14,
-          macos-13,
+          macos-latest,
+          macos-15-intel,
```

Comment:
the important bit here is to move away from `macos-13` as the image is deprecated (and possibly already discontinued). While I'm at it, I also bumped the amr64 image to match how ubuntu and windows are specified.